samples/secure-sql-chatbot/src/main/java/io/quarkiverse/langchain4j/sample/chatbot/MovieWatcherRepository.java

```diff
@@ -0,0 +1,16 @@
+package io.quarkiverse.langchain4j.sample.chatbot;
+
+import io.quarkus.hibernate.orm.panache.PanacheRepository;
+import jakarta.enterprise.context.ApplicationScoped;
+
+@ApplicationScoped
+public class MovieWatcherRepository implements PanacheRepository<MovieWatcher> {
+	
+	public String findPreferredGenre(String movieWatcherName, String movieWatcherEmail) {
+		MovieWatcher movieWatcher = find("name = ?1 and email = ?2", movieWatcherName, movieWatcherEmail).firstResult();
+        if (movieWatcher == null) {
+            throw new MissingMovieWatcherException();
```

Comment:
The problem with this is that this exception is swallowed and the bot just stops responding without error indication... I think this will need some better handling for content retriever errors in langchain4j, but until we have a proper fix, could we employ some workaround to make sure that the user gets an error? At least print the exception into the log here

Reply:
Ok I dug into it a bit more, and I think a solid solution would be to update the `ChatBotWebSocket.onMessage` method to catch and log an exception that is thrown from the `bot.chat(message)` call.

Reply:
Because if we don't catch the exception here, it's just swallowed and the UI will forever keep waiting for a response that never comes

Reply:
@jmartisk Jan, have a look please, I added 2 exception catch blocks